.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 20.8b1
```

Comment:
I'd rather not make use of pre-commit for anything in this repo.
The main reason why I dislike it is that it requires us to create yet another file where dependencies are defined. IMHO all dependencies should explicitly be defined in `pyproject.toml`.

Reply:
I see, ok I will remove it.